test/e2e/node_disconnect/lib/node_disconnect_lib.go

```diff
@@ -53,6 +54,25 @@ func (env *DisconnectEnv) ReconnectNode(checkError bool) {
 			Expect(err).ToNot(HaveOccurred())
 		}
 	}
+	env.nodeToIsolate = ""
+	env.disconnectMethod = ""
+}
+
+func SuppressMayastorPodOn(nodeName string) {
+	common.UnlabelNode(nodeName, engineLabel)
+	err := common.WaitForPodNotRunningOnNode(mayastorRegexp, namespace, nodeName, podUnscheduleTimeoutSecs)
+	Expect(err).ToNot(HaveOccurred())
+}
+
+// reconnect a node to the other nodes in the cluster
+func (env *DisconnectEnv) UnsuppressMayastorPod() {
+	if env.nodeToIsolate != "" {
+		common.LabelNode(env.nodeToIsolate, engineLabel, mayastorLabel)
+		// add the mayastor label to the node
```

Comment:
Should this comment be moved above line 70?

Reply:
Will do